src/main/java/hudson/plugins/claim/AbstractAssignedClaimsReport.java

```diff
@@ -0,0 +1,127 @@
+                ctx.setVariable("resURL", Stapler.getCurrentRequest().getContextPath() + Jenkins.RESOURCE_PATH);
+                return icon.getQualifiedUrl(ctx);
+            }
+        }
+        return null;
+    }
+
+    public String getUrlName() {
+        return "/claims";
+    }
+
+    @Restricted(DoNotUse.class) // jelly only
+    public Run getFirstFail(final Run r) {
+        Run lastGood = r.getPreviousNotFailedBuild();
+        Run firstFail;
+        if (lastGood == null) {
+            firstFail = r.getParent().getFirstBuild();
+        } else {
+            firstFail = lastGood.getNextBuild();
+        }
+        return firstFail;
+    }
+
+    @Restricted(DoNotUse.class) // jelly only
+    public CommonMessagesProvider getMessageProvider(final Run r) {
+        return CommonMessagesProvider.build(getAction(r));
+    }
+
+    @Restricted(DoNotUse.class) // jelly only
+    public CommonMessagesProvider getMessageProvider(final TestResult tr) {
+        return CommonMessagesProvider.build(tr.getTestAction(ClaimTestAction.class));
+    }
+
+    public ModelObject getOwner() {
+        View view = Stapler.getCurrentRequest().findAncestorObject(View.class);
+        if (view != null) {
+            return view;
+        } else {
+            return Jenkins.getInstance().getStaplerFallback();
+        }
+    }
+
+    private ClaimBuildAction getAction(final Run r) {
+        return ClaimUtils.getBuildAction(r, false);
+    }
+
+    public RunList getBuilds() {
+        List<Run> lastBuilds = new ArrayList<>();
+        for (Job job : getJobs()) {
+            Run lb = job.getLastCompletedBuild();
+            if (lb != null && isDisplayed(lb.getAction(ClaimBuildAction.class))) {
+                lastBuilds.add(lb);
+            }
+        }
+
+        return RunList.fromRuns(lastBuilds).failureOnly();
+    }
+
+	protected List<Job> getJobs() {
+		return Jenkins.getInstance().getAllItems(Job.class);
+	}
+
+    public List<TestResult> getTestFailuresWithClaim() {
+        List<TestResult> claimedTestFailures = new ArrayList<>();
+        for (Job job : getJobs()) {
+            Run lb = job.getLastCompletedBuild();
+            if (lb != null) {
+				AbstractTestResultAction testResultAction = lb.getAction(AbstractTestResultAction.class);
+				if (testResultAction != null) {
+		            List<? extends TestResult> failedTests = testResultAction.getFailedTests();
+		            for (TestResult failedTest : failedTests) {
+		                ClaimTestAction claimAction = failedTest.getTestAction(ClaimTestAction.class);
+		                if (isDisplayed(claimAction)) {
+		                    claimedTestFailures.add(failedTest);
+		                }
+		            }
+				}
+            }
+        }
+
+        return claimedTestFailures;
+    }
+    
+    protected boolean isDisplayed(AbstractClaimBuildAction<?> claimAction) {
+    	return claimAction != null && claimAction.isClaimed();
```

Comment:
Could this claimed-only by an option in the report ? I'm a little bit reluctant to remove a feature from existing uses, even if I admit that this might not be used at all). 
Would a checkbox with a default of not displaying unclaimed items fit ?

Reply:
If I correctly remember, the report currently displays builds with ClaimAction, no matter if they are broken or not and no matter if they are assigned or not. Using the same logic for unit tests with ClaimTestAction doesn't make sense to me as this would mean huge number of (hopefully green) unit tests.
I couldn't find a usefull logic to keep the "old way" and to allow displaying of the interesting Claim(Test)Action: the ones for broken builds & tests. If you have a better idea, I can try to help.

Reply:
Ok that's fair. If this is an issue to some users, we could reintroduce the feature only for builds.
Anoway it's not like if these screens would allow to claim unclaimed build/tests...